Add files via upload (#249)
sora-xor/sora-docs: src/adding-tokens-to-hashi-bridge.md

## Registering a SORA asset on the bridge

1. Get the AssetId of the asset you want to add (which you can find on [polkaswap.io](https://polkaswap.io)); e.g. `0x000268050a977248b641719592e7a0247ce4741839c83ec6aac6a865d3d0ba2c`).

2. Go to [polkadot{.js}](http://polkadot.js.org/) in the _Governance > Preimages_ section and create a preimage with `ethBridge.addAsset` and the value for the address and bridge network ID of the token to be registered. The parameters are `assetId` and `networkId` (`0` for Ethereum).

   ![](.gitbook/assets/add-asset-preimage.png)

> 3. Copy the preimage hash of the preimage you have just created and submit a proposal in _Governance > Democracy_. Alternatively you can try to reach out to a council member for the Council to directly create a motion to add the asset to the bridge with the preimage you created. This will fast track the process of adding the asset to the bridge.

Reviewer: ```suggestion
3. Copy the preimage hash of your created preimage and submit a proposal in _Governance > Democracy_. Alternatively, you can reach out to a council member for the Council to directly create a motion to add the asset to the bridge with the preimage you created. This will fast-track the process of adding the asset to the bridge.
```

> 4. Once the proposal goes through, acquire the bridge’s peers' signatures via the `ethBridge.getAccountRequests` RPC with your account as the first argument and status `ApprovalsReady`.

Reviewer 2: ```suggestion
4. Once the proposal goes through, acquire the signatures of the bridge’s peers via the `ethBridge.getAccountRequests` RPC with your account as the first argument and status `ApprovalsReady`.
```

   ![](.gitbook/assets/hashi-acquire-signatures.png)

> 5. The RPC will return a bunch of hashes. These are off-chain request hashes, the last one should be `OutgoingAddAsset` request. To find out if the asset was added successfully, use the `getRequests` RPC.

Reviewer: ```suggestion
5. The RPC will return a bunch of hashes. These are off-chain request hashes; the last one should be the `OutgoingAddAsset` request. To determine if the asset was added successfully, use the `getRequests` RPC.
```

> 6. After your hash is found, use `getApprovedRequests` RPC to get the approvals.

Reviewer: ```suggestion
6. After finding your hash, use the `getApprovedRequests` RPC to get the approvals.
```
